src/Pages/Home.js

```diff
@@ -67,7 +67,10 @@ class Home extends Component {
           twitterImage={metaData.metadataTwitterImage}
         />
         <Banner />
-        <LatestArticles language={this.props.language} />
+        <LatestArticles
+          articles={metaData.articles.value}
```

Comment:
please revert this loading
```suggestion
```

Reply:
@Simply007 I have reverted as suggested.